opentelemetry-sdk/CHANGELOG.md

```diff
@@ -64,6 +64,9 @@
  `LogExporter` trait no longer requires a mutable ref to `self`. If the exporter
  needs to mutate state, it should rely on interior mutability.
  [2764](https://github.com/open-telemetry/opentelemetry-rust/pull/2764)
+- **Breaking** for custom LogProcessor/Exporter authors: Changed `set_resource`
```

Comment:
It's only affecting `LogProcessor` authors, right?

```suggestion
- **Breaking** for custom `LogProcessor` authors: Changed `set_resource`
```

Reply:
true!